src/cli/dcos_docker/commands/create.py

```diff
@@ -195,6 +196,20 @@ def _validate_volumes(
     return mounts
 
 
+def _validate_loopback_device(
```

Comment:
Instead use `type=click.IntRange` on the option.

Reply:
Wouldn't I have to provide a max value in that case? Hm, probably could say something like `int.MAX` here.

Reply:
I believe `max=None` will work.